### DISCUSSION TITLE
zh/guide/markdown/include

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#445] [Announcements]
# zh/guide/markdown/include

让你的 VuePress 站点中的 Markdown 文件支持导入其他文件。

https://theme-hope.vuejs.press/zh/guide/markdown/include.html

**Comment by AndersonHJB** [comment]
@include(./demo.snippet.md#snippet) 能否导入网络中的纯文本文件呢？

@include(https://baidu.com/robots.txt)

**Reply by Mister-Hope** [reply]
实现不了，markdown渲染是同步操作
